### PR TITLE
Fix typos, improve wording in latest blog post

### PR DESCRIPTION
**Pull request by jplatte**

### CODE REVIEW

File: blog/_posts/2020-07-20-three-architectures-for-responsive-ide.adoc [hunk 1]
```diff
@@ -72,27 +72,27 @@ public class Main {
 
 The user has just typed `Foo.f().`, and we need to figure out that the type of receiver expression is `Bar`, and suggest `g` as a completion.
 
-First, as the `Main.java` file is modified, we run the indexer on this single file.
-Nothing has changed (the file still contains `Main` class with static `main` method), so we don't need to update FQN index.
+First, as the file `Main.java` is modified, we run the indexer on this single file.
```
Comment (jplatte) [inline blog/_posts/2020-07-20-three-architectures-for-responsive-ide.adoc:75]: The word swap here and in a few other places might not be required for grammatical correctness (I'm not actually sure), but it makes the article easier to read IMHO.

File: thisweek/_posts/2020-07-20-changelog-34.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@ Release: release:2020-07-20[]
 
 == Sponsors
 
-**Become a sponsor:** https://opencollective.com/rust-analyzer/[opecollective.com/rust-analyzer]
+**Become a sponsor:** https://opencollective.com/rust-analyzer/[opencollective.com/rust-analyzer]
```
Comment (matklad) [inline thisweek/_posts/2020-07-20-changelog-34.adoc:10]: whoops, excellent catch!

**Comment by lnicola** [comment]
bors isn't set up in this repo :smile:.

**Pull request merged**